doc/SUBSETS

```diff
@@ -1,7 +1,7 @@
 SUBSETS mode
 
 Subsets is a brute-force variant that tries to produce candidates in order of
-complexity, but without resorting to advanced stuff like Markov chains.  that
+complexity, but without resorting to advanced stuff like Markov chains.  That
 is, it will try a long poor password such as "gggggggggggggggggg#" much earlier
 than a short one with unique characters like "yok3" and in between them it will
 probably try "bubba" which is slightly longer but has repeated characters.
```

Comment:
Wouldn't it try "bubba" before "gggggggggggggggggg#"? Perhaps just drop the "#" from the first example so that it's actually tested before "bubba".

Reply:
True, and also it will actually *never* try "gggggggggggggggggg#" unless you use `--max-len=19` to bump max. length from the default of 16... but I did guard the example with the word "probably" 🤣 so I'll just let it be for now. Hopefully the reader gets the general idea.